gensim/models/word2vec.py

```diff
@@ -641,16 +678,16 @@ def scan_vocab(self, sentences, progress_per=10000, trim_rule=None):
             if sentence_no % progress_per == 0:
                 logger.info(
                     "PROGRESS: at sentence #%i, processed %i words, keeping %i word types",
-                    sentence_no, sum(itervalues(vocab)) + total_words, len(vocab)
+                    sentence_no, total_words, len(vocab)
                 )
             for word in sentence:
                 vocab[word] += 1
+                total_words += 1
```

Comment:
This is not a good idea, will be slow. Add the entire `len(sentence)` at once.

Reply:
hmmm although it wont noticeably affect the speed, but yes it should be incrementing at once :+1: